src/algebra/group_power/lemmas.lean

```diff
@@ -83,6 +83,23 @@ begin
   exact is_unit_of_invertible x
 end
 
+lemma smul_pow [monoid N] [mul_action M N] [is_scalar_tower M N N] [smul_comm_class M N N]
```

Comment:
```suggestion
lemma smul_pow [mul_action M N] [is_scalar_tower M N N] [smul_comm_class M N N]
```
I would guess you can fold the line wrapping too as a result

Reply:
Yup, I hope to get to this after my workday =)